.github/workflows/gene_update.yml

```diff
@@ -0,0 +1,37 @@
+name: Updated Genes
+
+on:
+  pull_request:
+    paths:
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/gene_info.yml'
+
+jobs:
+  gene-processing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Python cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: gene-processing
+        run: |
+          pip install -r cellxgene_schema_cli/requirements.txt
+          make gene-processing -C cellxgene_schema_cli
+      - name: Commit
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add .
+          git commit -m "AUTO: update genes"
+          git push
```

Comment:
Same note as the other PR--push tests don't run in the PR after this commit, which could hide failing tests